src/main/java/minijava/ir/DefaultNodeVisitor.java

```diff
@@ -0,0 +1,170 @@
+package minijava.ir;
+
+import firm.nodes.*;
+
+public interface DefaultNodeVisitor extends NodeVisitor {
```

Comment:
I like to have a NodeVisitor interface. I'm not happy with using an abstract class.